Assets/Scripts/Controllers/CharController.cs

```diff
@@ -43,7 +43,6 @@ protected enum CharacterDir
     protected void Start()
     {
         animator = GetComponentInChildren<Animator>();
-        currentHealth = maxHealth;
```

Comment:
I see this logic is being moved from the parent to the child classes, but I don't understand why

Reply:
Because currentHealth is only set to maxHealth on the player when there is a save file. The enemies always start with mnax health, obviously, so i had to break it apart.

Reply:
Right, that makes sense